src/main/java/io/github/dsheirer/module/log/EventLogger.java

```diff
@@ -75,7 +73,7 @@ public void start()
                 sb.append(File.separator);
                 sb.append(TimeStamp.getTimeStamp("_"));
                 sb.append("_");
-                sb.append(replaceIllegalCharacters(mFileNameSuffix));
+                sb.append(StringUtils.replaceIllegalCharacters(mFileNameSuffix));
```

Comment:
Can you please move the cleaning method up to the ChannelProcessingManager:290 and clean the channel.getName() value.  The current approach causes path separators in linux to be converted to underscores and causes an illegal filepath.


        /* Setup event logging */
        List<Module> loggers = mEventLogManager.getLoggers(channel.getEventLogConfiguration(),
            StringUtils.replaceIllegalCharacters(channel.getName()));